docs/developer/plugins.md

```diff
@@ -108,16 +108,43 @@ You can then set this option in your config file:
 
 ```groovy
 // dot syntax
-myplugin.create.message = "I'm alive!"
+myplugin.createMessage = "I'm alive!"
 
-// closure syntax
+// block syntax
 myplugin {
-    create {
-        message = "I'm alive!"
+    createMessage = "I'm alive!"
```

Comment:
What about structured options e.g. `wave.build.repository` ?

Reply:
You can see how I handled that [here](https://github.com/nextflow-io/language-server/blob/main/modules/compiler/src/main/java/config/scopes/WaveConfig.java)

Reply:
Ok, so it can handle nested objects